converter/internal/prometheusconvert/prometheusconvert.go

```diff
@@ -49,38 +42,96 @@ func Convert(in []byte) ([]byte, diag.Diagnostics) {
 		diags.Add(diag.SeverityLevelError, fmt.Sprintf("failed to render Flow config: %s", err.Error()))
 		return nil, diags
 	}
-	return buf.Bytes(), diags
+
+	if len(buf.Bytes()) > 0 {
```

Comment:
Thoughts on inverting this to `if len(buf.bytes) == 0 { return nil, diags}`, that would match the pattern above of the ifs enclosing the non-happy path